php/media/class-gallery.php

```diff
@@ -237,14 +237,7 @@ public function block_editor_scripts_styles() {
 			true
 		);
 
-		wp_localize_script(
-			'cloudinary-gallery-block-js',
-			'cloudinaryGalleryApi',
-			array(
-				'endpoint' => rest_url( REST_API::BASE . '/image_data' ),
-				'nonce'    => wp_create_nonce( 'wp_rest' ),
-			)
-		);
+		wp_add_inline_script( 'cloudinary-gallery-block-js', 'var CLD_REST_ENDPOINT = "/' . REST_API::BASE . '";' );
```

Comment:
Is it OK that this gets added _after_ the `cloudinary-gallery-block-js` is loaded? Shouldn't it be before the script which uses this data? https://developer.wordpress.org/reference/functions/wp_add_inline_script/

Reply:
This works because the variable will be defined by the time it's needed, but your suggestion adds proper precautions for when we'd need the variable at an earlier point. Thanks!